site/installing.md

```diff
@@ -236,7 +236,7 @@ Allowable parameters for the launcher URL:
 
 To download and read the Scope manifest run:
 
-    curl --silent --location --remote-name https://cloud.weave.works/k8s/scope.yaml
+    curl --silent --location --remote-name "https://cloud.weave.works/k8s/scope.yaml?k8s-version=$(kubectl version | base64 | tr -d '\n')"
```

Comment:
TBH, I would consider dropping this. Folks who want to look at the yaml should be capable of constructing a curl themselves. It will take a couple of tries due to the redirect, but it's not exactly hard.